wolfcrypt/src/asn.c

```diff
@@ -1027,7 +1027,7 @@ static int GetASN_Integer(const byte* input, word32 idx, int length,
     (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION > 2))
     /* Check contents consist of one or more octets. */
     if (length == 0) {
-    #ifdef WOLFSSL_DEBUG_ASN_TEMPLATE
+    #ifdef WOLFSSL_DEBUG
```

Comment:
Is WOLFSSL_DEBUG a new macro or typo? I see a hit on it at a couple spots with grep. With using WOLFSSL_MSG though the macro guard maybe could be removed here, or changed to guard on WOLFSSL_ASN_INT_LEAD_0_ANY.

Reply:
Should have removed the ifdef altogether.
Reworked to minimal change.